inbm/dispatcher-agent/tests/unit/test_dispatcher.py

```diff
@@ -101,7 +101,30 @@ def test_pre_check_pass(self,
     @patch('inbm_lib.mqttclient.mqtt.mqtt.Client.subscribe')
     @patch('dispatcher.dispatcher_class.Dispatcher._send_result')
     @patch('dispatcher.dispatcher_class.Dispatcher.invoke_workload_orchestration_check')
-    def test_aota_thread_start_called(self,
+    def test_aota_thread_start_called_without_signature(self,
+                                      mock_workload_orchestration_func: Any,
+                                      mock_send_result: Any,
+                                      m_sub: Any,
+                                      m_connect: Any,
+                                      m_thread_start: Any,
+                                      mock_logging: Any) -> None:
+        xml = '<?xml version="1.0" encoding="UTF-8"?>' \
+              '<manifest><type>ota</type><ota><header><id>sampleId</id><name>Sample AOTA</name><description>' \
+              'Sample AOTA manifest file</description><type>aota</type><repo>remote</repo>' \
+              '</header><type><aota name="sample.rpm"><cmd>load</cmd><app>docker</app><fetch>http://www.example.com/</fetch>' \
+              '<version>1.0</version><containerTag>defg</containerTag><signature>abcdefg</signature><sigversion>384</sigversion>' \
+              '</aota></type></ota></manifest>'
```

Comment:
**Suggestion:** The XML string used in the test 'test_aota_thread_start_called_without_signature' is hardcoded and repeated. Consider refactoring to use a helper method or a fixture to create the XML string to improve maintainability and reduce code duplication. [enhancement]
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        xml = self.create_aota_manifest(sample_id='sampleId', name='Sample AOTA', description='Sample AOTA manifest file', 
                                        repo='remote', package_name='sample.rpm', cmd='load', app='docker', 
                                        fetch_url='http://www.example.com/', version='1.0', container_tag='defg', 
                                        signature='abcdefg', sigversion='384')
```

Reply:
this is test code--some repetition is fine for clarity